src/xmipp/libraries/reconstruction/directions.h

```diff
@@ -27,7 +27,7 @@
 #define _DIRECTIONS_H
 
 #include <core/xmipp_funcs.h>
-#include <core/metadata.h>
+#include <core/metadata_vec.h>
```

Comment:
where possible, try to move includes to cpp files

Reply:
I understand. Here, include is required. TODO discuss.